galata/test/documentation/extension_manager.test.ts

```diff
@@ -81,6 +86,8 @@ test.describe('Extension Manager', () => {
     expect(
       await page.screenshot({ clip: { y: 31, x: 0, width: 283, height: 600 } })
     ).toMatchSnapshot('extensions_search.png');
+
+    await unstubExtensionsSearch(page);
```

Comment:
Yeah, it turns out `page.route` doesn't remove the route stub after a test completes, so it's required to call `page.unroute` manually after each test. If anybody knows an alternative to this (that doesn't make it so easy for devs to accidentally share stubs across tests), please LMK!

Reply:
@dlqqq how did you come to that conclusion? The `.route` is linked to the page object that is unique for each test (except when using some specific describe block). So I'm a bit surprised it needs to be undone explicitly.

Reply:
I noticed this when adding `page.route` caused the other snapshot tests to fail. I'm also surprised, since I also assumed that the page object is unique per test. This probably warrants further investigation since ideally page routings shouldn't get shared across tests.